### DISCUSSION TITLE
in prep for Week 6: first project iteration!

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#8] [Show and tell]
Hi, everyone! You can **use this space to post your 5-minute videos**, along with a link to your most current [public-facing project deliverable](https://benmiller314.github.io/dsam2023fall/projects#palimpsest-of-public-iteration); we'll then be able to use the discussion forum to [write replies](https://benmiller314.github.io/dsam2023fall/projects#presentations-and-peer-review), too.

<div class="alert alert-success"><p>To upload a video, just drag and drop the file onto your "Write" tab. <a href="https://youtube.com/clip/UgkxLpJlcixbtxe4iJdDXwh8nyHj8Oh3-Q3D?si=swQMcoUIptTxbrq7">It's that easy!</a></p></div>

Just so you know (and I hope this puts you further at ease), I will *not* be requiring that you play your video in class while you listen to your own recorded voice and squirm. Instead, because you will likely have to practice to get your presentation to fit in 5 minutes, you'll be well-rehearsed to give your 5-minute presentation live to the class, leaving us plenty of time to have a conversation about your work.

See the [end of week 4's lesson plan](https://benmiller314.github.io/dsam2023fall/plans/week-04#hw) for more detailed guidance, but essentially for this round of presentations **we're just looking for a teaser,** an entry-point into your project. Quick, quick: What are you trying to do? What have you put together so far? What are you excited about? What are you wondering? And out. Like one minute each. :¬)

**Comment by SidraNArshad** [comment]
https://github.com/benmiller314/dsam2023fall/assets/142813292/5b569cd1-963f-4089-a4ed-38dee9cefb72

https://scalar.usc.edu/works/dsam-project---blm-activists-/index

**Reply by amj169** [reply]
Peer Review of Sidra’s Project:

Great work here, Sidra! This is an important project and I’m excited to see where you take this influential work. 

What is it/What are you asking?  I understand that you are tracing narratives of BLM activists who have passed. You’re interested in tracing parts of their lived experiences before and after their deaths. What were their stories before and how are they being discussed/carried on after?

Public Facing Product- Developing a website to house such content

Questions:
-Pre-work: Tried to peruse stories that were available surrounding Ferguson murders. Where did you look? Are you looking in certain areas? 
-Website is set up so the victims are the first thing you see on the page. You do a nice job taking care in the ways you present this the victims’ images. You consciously aimed to forefront the act of maintaining personhood instead of the pictures associated with their deaths. 

RE: exploring social media accounts before and after their deaths. Were all of these people involved in other types of activism? In their communities? This might be another way to help the story unfold beyond their deaths.

Where did you find these photos? Social media I wonder if you’ve considered reaching out to their families as a courtesy to discuss them publicly. Obviously this isn’t the same thing, but I remember when Netflix released DAHMER, they apparently didn’t reach out to the families of the victims which resurfaced an unimaginable amount of trauma. I genuinely don’t know the answer here, just a thought!

-Has a page of recurring things that played roles in their deaths. Hmm. She is wondering how to link/discuss recurring themes…How do I do this best while maintaining their narratives without taking this away their personhood? Ben mentioned link news articles as a means to do that work for you.

I also wonder if you have looked at any similar work about indigenous communities or school shooting victims to see how others may be discussing things of this sensitive nature. Can you look to other public facing projects? You also might look to poems or writings to categorize this. Again, just thinking about how you might bring in other tools to help do some of the delicate and difficult pieces of this carefully for you. You might be interested in checking out Jackie Wang’s [Carceral Capitalism.](https://www.are.na/block/5227591) I’m specifically remembering a chapter where she talks about Ferguson, Mike Brown and Eric Garner in A Dozen Roses Against the Police State. She does a lot to lift the voices of those who were murdered in the legal system. 

 If you want to explore this you might also visit [CAAP’s website.](https://www.caapp.pitt.edu/) The theme last year was “Close Reading the Emergency” and there was a lot of really cool stuff.  

Where Are you tracing? How did you decide who to include? Certain area or timeframe? Began in 2014 with Ferguson – then started looking from there.

Do you have any info about their communities? Hobbies? Family life, maybe? To do this I wonder if you look and see if there is a legacy or foundation that was left in memory of them. That might help promote them in a way that pays tribute or carries their memory rather than just being numbers claimed by racial violence.

-In terms of presentation, what does this look like? Will you partner with someone, an organization, the families, to share this? Have you looked into any anti-racist journals? Calls for papers/workshops/conferences/symposiums? I would guess Pitt has something you can submit this to. I’m VP of the Cultural Studies & Gender, Sexuality, and Women’s Studies grad student organization and we usually run a symposium in the spring to host student work. I will give you more details if you’re interested. 

 Thanks for sharing and nice work!!

**Reply by chloe-cmu** [reply]
Sidra's Project Peer Review

What you're doing (as I understand): comparing the social media presences of BLM activists with media coverage of them after their deaths, to see what you can uncover about how the narratives relate to each other (or don't). 

I think this has the potential to be a really impactful and moving project. I do have some questions you might want to think about as you continue to build it. 

1. I want to second Alex's point about reaching out to the families. As you're tracking multiple stories of intense dehumanization, returning agency to victims and their families is important. Of course, that's a whole endeavor on its own, so perhaps you could do a demo version of the website privately and then make it public if/when you have permission? Then you could also have something to show survivors to explain what you're trying to do. 
2. Right now, it seems to me that this project is casting a pretty wide net in terms of what it's looking at. Have you considered the possibility that listing unrelated BLM activists who have died may support another conspiracy theory along the lines of the one that grew up among activists after Ferguson?  Grouping people together in a public forum, even if they never encountered each other in life, can create an appearance of deeper connection. That's not to say you can't do it, it just might bear some thought. 
3. Also in terms of the wide net considerations: if you wanted to limit the scope of the project a bit, I think you could actually limit it to Ferguson, because I think there are a few narratives there that could be interesting to look at. 1 - the initial coverage of the protests; 2 - activists lives; 3 - activist deaths; 4 - the murder conspiracy theory; 5 - any potential coverage of Ferguson during the George Floyd protests. That's a big project in a different way, but I just wanted to mention it as an idea in case you wanted to tighten the frame a little bit or if it sparked another idea. 

The project looks really good so far, and no matter what direction you go, I think it will be important. Thank you for sharing!

**Comment by Khushboobhutani** [comment]
https://github.com/benmiller314/dsam2023fall/assets/143055269/c4113e9f-5f29-43a3-9c4f-a2ae5e6c2fb0

**Reply by Khushboobhutani** [reply]
https://scalar.usc.edu/works/dsam-project-tracing-israeli-palestinian-conflict

Here is the link to my project on Scalar

**Reply by amj169** [reply]
Peer Review of Khushboo’s Project

You have a lot of great material here, Khushboo! I really enjoyed listening to your presentation and I’m excited to see which direction you decide to take this.

What is it/What are you asking?

How technology is used to navigate conflict specifically war activism in Israel and Palestine

- focusing on video essays about these events... but perhaps not enough data to support this inquiry
I’m wondering who is generating this content? 

What stakes do the people who are generating these videos have? Are they directly impacted? Are there people advocating/protesting from a distance? Government parties/lobbyists?

Will use different maps as timelines of activism in Palestine. 1948… but decided not to do this. Too out of scope. Just looking at reviews of conflict of 2021

Are you only looking at videos of people who are talking about this in Palestine or people who might be involved outside of Palestine? 
Will these act as context for your present day analysis or will you use news articles and investigate how the discourse has changed over the years? I was wondering this before it became clear that you decided not to focus on older stuff from the 1940s.

I think it will be interesting to see how mediums have changed or shifted even just in 2021 to publish this information. I wonder what common characteristics of videos on one platform vs another may differ. Are you looking at Tik tok? Is the content different than a video essay or Instagram or Facebook reel? You might be able to get an idea about how the rhetorical action of conflict reporting changes based on consumption and circulation. 

What shifts do you think you’ll find when you look at content across different mediums? 
How will you categorize these shifts or differences in discourse surrounding the videos? Using them for activism v propaganda coming from outside sources/special interests. 

I wonder if you have a history page-shifting nature of protests? 

Will you look at the comments on the videos to determine how people are receiving the media? 

I also wonder if you might make a video to post on your webpage explaining how you made the map/explaining where you’d like the user to begin looking to help the user navigate it. I have ADHD, so sometimes I have a hard time with interactive sites and knowing where to begin because I’m easily distracted. Just a thought! 

Are you thinking about submitting this to a journal call for papers/conference/workshop/symposium? If so, which ones are you considering?

Thanks for sharing and nice work!! 
Alex

**Reply by Rezaetmd** [reply]
First of all, I really appreciate your work on the subject as it can open up a social space for discussion for people who have no information about Palestinian lives. I also enjoyed your presentation as everyone else did! I have a few questions for you: 

1. Now that you’re gathering data from multiple sources (and there are obviously many), do you have any idea how you’re going to prioritize your data?
2. I also think that you should decide at some point what sources are more beneficial for your project since there are some that engage more deeply (and realistically) with what’s happening in the region, and some which contain mostly misinformation. For example you might find convoluted accounts aimed at a certain age group in TikTok. Would you consider them as reliable info and include them in your data collection? 
3. Would you consider to put up your project as a website on the web? 

Look forward to see how your project unfolds through the coming weeks, and my best wishes,
Reza

**Comment by amj169** [comment]
Alexandria Jennings shared a CapCut link with you: https://www.capcut.com/view/7285215201404649990?workspaceId=7284430750529929222

Here is the link to my video. It wouldn't let me drag and drop it here. Apparently it's too big. If the link doesn't work for some reason, I can just play it on my laptop.

**Reply by amj169** [reply]
[phd stipend data tidied.csv](https://github.com/benmiller314/dsam2023fall/files/12783857/phd.stipend.data.tidied.csv)

[PhD stipend before tidy.csv](https://github.com/benmiller314/dsam2023fall/files/12784163/PhD.stipend.before.tidy.csv)

Data origins:
https://www.phdstipends.com/faq
https://carnegieclassifications.acenet.edu/about/faqs/
https://writingcensus.ucsd.edu/

**Reply by SidraNArshad** [reply]
Peer Review Notes:

In terms of repeating back to you what I think your project is, I want to say that your research is focused on understanding the trends that exist in PhD stipends for programs specific to the U.S. and in the field of English...? I know you're at that stage in your project where you are just trying to make sense of your content, and when that is done, you'll be able to move toward how you want to categorize the info you have and potentially make comparisons across institution types. Some questions that I have for you:

Do you plan on categorizing the institutions by region?
How do you see yourself translating the data that you have into something visually accessible?
Are you going to be contextualizing stipends in terms of the cost of living?

**Reply by MustafaKandil** [reply]
Hello Alex,

I wanted to express my appreciation for the great presentation you did on Ph.D. stipend rates in the USA, especially considering it's such a relevant topic for us Ph.D. students!

I was particularly impressed with your creative introduction; it really captured my attention. From what I gathered, you're utilizing Carnegie data as your primary source and comparing its accuracy with data from other institutions. Additionally, you mentioned your intention to categorize this dataset. I'm curious, will this categorization be done for every PhD program, and if so, could you share some insights into the categorization model you plan to employ? Are you considering categorizing by region, school, or department?

Your project is undeniably exciting, and I'm eagerly looking forward to its development. Best of luck with your ongoing work.

Sincerely,

**Comment by MustafaKandil** [comment]
https://github.com/benmiller314/dsam2023fall/assets/143913512/182b888a-a01d-4492-8e4b-ff16f415cf56

**Reply by chloe-cmu** [reply]
Hi Mustafa 

Your project (As I understand it) - You want to use ArcGIS to map UNESCO's list of Intangible Cultural Heritages in Turkey, to demonstrate the ethnic and cultural diversity of Turkey. 

Just a few questions to consider as you work on this: 
1. How does UNESCO determine what intangible cultural heritages are and where they are? I know you just want to make a map of the list, but I think it's important to figure out how things end up on that list, especially when the list is written by a supranational organization, rather than Turkish people directly. 
2. Do you plan to represent specific traditions in different ways? The list of intangible cultural heritages seems pretty broad, from social practices to festivals to making traditional crafts. Have you thought about representing this diversity with different colors on the map? Perhaps you could group them on your own - like one category for performances, another for making food or crafts, etc. 
3. Do you plan to include information about if particular traditions are practiced by particular ethnic subgroups? Maybe you could think about having a filter or a layer so a viewer could see all of the intangible cultural heritages practiced by Kurds, for example. 

This looks like a super cool project, and I'm excited to see where it goes!

**Reply by Rezaetmd** [reply]
Hi Mustafa,

I also agree with Chloe on reframing your thoughts about using UNESCO list as your data source. I know that you wanted to make sure that you're making a 'circle' here in a way to decide what cultural processes and events are included in your project but you can also refer to probably some books or other sources and get a more diverse list of those events, possibly some that are not included by UNESCO. Also, I think you can be benefited from some field works (even if it means some short interviews with the people in the region) to prioritize which cultural heritages you're going to include in your project.

Looking forward to your project,
Good luck!

**Comment by chloe-cmu** [comment]
[DSAM Presentation 2023-10-2.zip](https://github.com/benmiller314/dsam2023fall/files/12782780/DSAM.Presentation.2023-10-2.zip)

**Reply by chloe-cmu** [reply]
csv file here with current project status: https://github.com/chloe-cmu/foodadditives

**Reply by Khushboobhutani** [reply]
Hi Chloe

Thank you for your presentation. It was indeed revelatory. 

Your research examines the origins and uses of food additives in the US in order to make this information more legible to scholars. You used Open Refine to categorize the FDA data into four categories which will quite helpful to understand and weigh the varied implications underlying the use of such additives. 

I am wondering if your project is addressed only to scholars or do you wish to make it legible for the larger public. In doing so, it might require you to introduce a few more parameters of division or maybe something that alerts people with regards to the hazards associated with particular additives. 

Do you plan to make your data more interactive given the substantial amount of information you are dealing with? 

Do you wish to experiment with other modes of data collection and translation other than Open Refine?

**Reply by MustafaKandil** [reply]
Hello Chloe,

Thank you for delivering such an excellent presentation.

I totally agree with Khushboo's sentiments regarding your project's objectives. Your explanation of the topic and its goals was clear and concise. Additionally, I was interested by the idea of delving into the historical background of food additives and tracing their evolution over the years.

If I understood correctly, you mentioned that publicly accessible data on this topic can be messy and inconsistent. I'm curious to know if your aim is to make this data more accessible exclusively for scholars or if you intend to create a well-organized and consistent data source that can benefit researchers and scholars alike. The latter proposition, of establishing an organized and open-source data repository, holds great promise for the academic community.

Your project is indeed exciting, and I look forward to its development. Best of luck with your endeavor.

Sincerely.

**Comment by SidraNArshad** [comment]
Peer Review Notes:

Reza's Project:

As far as I understand it, your project is focused on how the genre of farsifilms was ignored and denigrated by three critics from the 1950s-70s, as well as how this genre had an effect on popular tastes during that time period? Or, the popular tastes of today? I know that you're looking through different books and magazines for information, do you plan on watching any of the films in order to give you a foundation for what the critics are talking about? 

1. How are you going to track the effect of these movies on popular tastes?
2. Have you figured out how many movies fall under the “Farsifilm” category?
3. Will you be looking at all of them? Some of them? How will you decide?
4. How are you planning on presenting this information visually on website? I think it would be cool to incorporate some short clips from the films, but you would have to consider how available the films are to edit them like that and I'm not sure how copyright would make its way into that.

**Comment by Khushboobhutani** [comment]
Peer review for Reza's project

Thank you for the engaging presentation on Oct 2. Your project elaborates on _Filmfaarsi,_a category of films denounced by Iranian critics during the mid 20th century. Based on our discussions in the classroom, I would encourage you to think about your methodology and categorize your data accordingly considering the enormity of it. 

Are you planning to examine the sales and distribution logic of such films so that one can understand how were the audiences responding to this brand of cinema? Were the critics' opinions limiting them or did it have no effect? Did the critical opinions only end up being a theoretical category eventually? 

In continuation of this, would you also include the anecdotes from the audience who enjoyed such films irrespective of their no overt social message? To this end, will you also include some film clips in your presentation?

Talking about how to problematize the idea more, do you plan to discuss the afterlife of such films or maybe include their posters, advertisements, dvd covers etc? If there is an online archive of these videos, do people access it regularly and enthusiastically? 

All the best!

**Comment by Rezaetmd** [comment]
**Here is a short intro to my project:**

What are you trying to do? 

I’m trying to examine early Iranian cinema and answer this question: how a particular argument related to low-budget films was formed in the history of Iranian cinema through writings of three main critics from the 1950s to the 1970s? And also, what relationship does it have with the cultivation of popular taste at that time? The sources of the project would be film magazine archives, books and still frames of some of the films generally classified as Filmfarsi. While there are few academic articles and books that have attended to this subject, I hope that by visualizing the material and rephrasing them in a way to make them less esoteric along with including some other contents from old magazines, give general audiences an introduction into this long-standing debate. 


What have you put together so far? 

I have gathered and read sections of books and magazines about early Iranian cinema. I have chosen and translated some as my project references. I have also been working with Scalar for the past week and tried to put together a few pages in a ‘Book’ in it.

What are you excited about? 

I’m excited to see how the project ends up in. I also want to try OCRing some Persian magazines and see if I can search through them instead of looking for stuff by myself. I have tried Google Drive so far and it turned out to be useful for some but not all of Persian texts.

**Reply by Khushboobhutani** [reply]
Peer review for Reza's project

Thank you for the engaging presentation on Oct 2. Your project elaborates on _Filmfaarsi,_a category of films denounced by Iranian critics during the mid 20th century. Based on our discussions in the classroom, I would encourage you to think about your methodology and categorize your data accordingly considering the enormity of it.

Are you planning to examine the sales and distribution logic of such films so that one can understand how were the audiences responding to this brand of cinema? Were the critics' opinions limiting them or did it have no effect? Did the critical opinions only end up being a theoretical category eventually?

In continuation of this, would you also include the anecdotes from the audience who enjoyed such films irrespective of their no overt social message? To this end, will you also include some film clips in your presentation?

Talking about how to problematize the idea more, do you plan to discuss the afterlife of such films or maybe include their posters, advertisements, dvd covers etc? If there is an online archive of these videos, do people access it regularly and enthusiastically?

All the best!
